docs/configuration-files.rst

```diff
@@ -54,71 +54,86 @@ It defines and documents all supported sections and options. The file is located
 .. include:: ../data/anaconda.conf
     :code: ini
 
-Product configuration files
+Profile configuration files
 ---------------------------
 
-The product configuration files allow to override some of the configuration options for
-specific products. The files are located at ``/etc/anaconda/product.d/``.
+The profile configuration files allow to override some of the configuration options for
+specific profiles and products. The files are located at ``/etc/anaconda/profile.d/``.
 
 .. note::
 
     Anaconda previously used so called install classes for the product-specific configuration.
-    Install classes were completely removed and replaced by the product configuration files.
+    Install classes were completely removed and replaced by the profile configuration files.
+    These configuration files used to be called product configuration files for some time.
```

Comment:
I would remove this sentence. From the new user perspective is this information not valuable and for older users used to InstallClasses it will not be helpful too.

Reply:
It could be valuable for current users. The install classes were replaced by product configuration files in January 2019.